Source/DafnyCore/AST/DafnyAst.cs

```diff
@@ -77,108 +78,114 @@ public abstract class INode {
     // TODO: Re-add format tokens where needed until we put all the formatting to replace the tok of every expression
     internal IToken[] FormatTokens = null;
 
-    public RangeToken RangeToken {
-      set => rangeToken = value;
-    }
+    [DebuggerBrowsable(DebuggerBrowsableState.Never)]
+    public virtual RangeToken RangeToken {
```

Comment:
I think this should only be a property if we plan to remove the computation. Could you link from here to an issue that describes the change you have in mind?

Reply:
Right, that's exactly the plan that, following your advice, I described here:
https://github.com/dafny-lang/dafny/issues/3290